code/scpui/elements/ScrollingTextElement.cpp

```diff
@@ -56,7 +56,8 @@ void ScrollingTextElement::OnChildRemove(Element* child)
 void ScrollingTextElement::OnUpdate()
 {
 	if (_newAnimationStarted) {
-		gamesnd_play_iface(InterfaceSounds::BRIEF_TEXT_WIPE);
+		//Disabled because this should be controled by the Lua to prevent it playing on frame 0
+		//gamesnd_play_iface(InterfaceSounds::BRIEF_TEXT_WIPE);
```

Comment:
Ah I see. This will leave an empty method behind because it seems like librocket expects OnUpdate to exist.

Reply:
If you delete the method both from ScrollingTextElement.h and .cpp, it should be fine. But IMO more important is that you delete _newAnimationStarted in ScrollingTextElement.h and all its mentions in the cpp, since they aren't needed anymore